Tools/msgstringtable/msgstringtable_decoded.txt

```diff
@@ -2976,8 +2976,8 @@ MSI_SUCCESS,"Success"
 MSI_RESET,"Return"
 MSI_MANUFACTURED_MARRIAGERING_ITEM,"With Mr/Miss"
 MSI_CHANGE_HOMUN_NAME,"The Homunculus's name will be changed to^0000ff^0000ff %s^000000^000000, Are you sure?"
-MSI_BLOCK_CALL_PERMIT,"Call function is ON"
-MSI_BLOCK_CALL_REJECT,"Call function is OFF"
+MSI_BLOCK_CALL_PERMIT,"You can be \"Called\" with various skills (Emergency Call,...). (/call)"
```

Comment:
Personally I'd go with "Player recalling skills is disabled/enabled" same for the bottom related call messages.

Reply:
I think it’s for this way, as it’s only affecting the player not the skills in general.